src/integrations/prefect-aws/prefect_aws/credentials.py

```diff
@@ -107,6 +109,16 @@ class AwsCredentials(CredentialsBlock):
         description="Extra parameters to initialize the Client.",
         title="AWS Client Parameters",
     )
+    role_arn: Optional[str] = Field(
```

Comment:
Could we name this `assume_role_arn` for some consistency between this field and `assume_role_kwargs`?
```suggestion
    assume_role_arn: Optional[str] = Field(
```